feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.spec.ts

```diff
@@ -448,4 +456,27 @@ describe('ConfigAddToCartButtonComponent', () => {
       });
     });
   });
+  describe('Floating button', () => {
+    const observerMock = {
+      observe: () => null,
+      isIntersecting: true,
```

Comment:
You only test the case intersection equals true, can you add the false case as well.